### ISSUE TITLE
'{"errors":[{"message":"Sorry, that page does not exist","code":34}]}'

### ISSUE THREAD

**Issue by isrmicha** [the-convocation/twitter-scraper#7]
Anyone getting this trying to use scrapper.getTweets ?

**Comment by karashiiro** [comment]
The scheduled CI task last ran [3 days ago](https://github.com/the-convocation/twitter-scraper/actions/runs/5395285955) and passed successfully, so this is either a new issue or it's something that the tests aren't catching. Can you share a brief repro with the failing parameters?

**Comment by isrmicha** [comment]
```

(async ()=> {
    const { Scraper } = await import("@the-convocation/twitter-scraper");
    const scraper = new Scraper();
    const tweets = scraper.getTweets("uecaiu", 10, false);
    for await (const tweet of tweets) {
      console.log(tweet);
      }
})()
```

**Comment by isrmicha** [comment]
can you try run the CI again ?
in my fork it fails
https://github.com/isrmicha/twitter-scraper/actions/runs/5425921982/jobs/9867323086

**Comment by isrmicha** [comment]
![image](https://github.com/the-convocation/twitter-scraper/assets/8812005/0ba61c01-58b2-42a9-88cb-07bc72c70383)

**Comment by Owen3H** [comment]
I'm not able test at the current moment, but theoretically using `login()` in TwitterUserAuth should fix this?

**Comment by Owen3H** [comment]
Upon inspection, it seems necessary for all functions to require auth similar to search, at least temporarily.

**Comment by karashiiro** [comment]
This would also be fixed once https://github.com/n0madic/twitter-scraper/commit/fb8ae789bd973266ca2b36b624b3f67124758ce8 is ported over, whichever comes first.

**Comment by flyfoxuk** [comment]
I think this latest problem with adaptive.json no longer in use needs the fix here that implements the GraphQL endpoint for searches: https://github.com/n0madic/twitter-scraper/issues/116

**Comment by karashiiro** [comment]
This is fixed as of [v0.3.0](https://github.com/the-convocation/twitter-scraper/releases/tag/v0.3.0).
